tests/yang/YANGBearTest.py

```diff
@@ -0,0 +1,24 @@
+import requests
+
+from bears.yang.YANGBear import YANGBear
+from coalib.testing.LocalBearTestHelper import verify_local_bear
+
+
+VALID = requests.get(
+    'https://raw.githubusercontent.com/OpenNetworkingFoundation/CENTENNIAL'
+    '/master/models/yang/core-model.yang'
+).text
+
+
+# several missing module sub-statements
+INVALID = """
```

Comment:
There will eventually be more than one invalid file. Please give this variable a useful name.